.github/workflows/ci.yaml

```
@@ -0,0 +1,52 @@
+# Copyright 2024 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+name: cephfs-server-proxy tests
```

Totally missed this on the first pass :rofl: - I'll commit the suggestion since you're enjoying the weekend!

```suggestion
name: hpc-libs tests
```